.github/ISSUE_TEMPLATE/03-reportmissinggame.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Missing Game
+about: Report a game that was not correctly detected by the app
```

Comment:
I think it's important to include "supported" somewhere in here. This template is for issues with the app not detecting _supported_ games.

Reply:
Added this to the summary and template.